package.json

```diff
@@ -188,6 +188,7 @@
 		"run-script-os": "^1.1.6",
 		"syncpack": "^13.0.2",
 		"ts2esm": "^1.4.0",
+		"type-fest": "^2.19.0",
```

Comment:
This dep seems to be leaking through via build-tools which I don't understand, but this root dep resolves it and I don't think it's worth chasing down right now.